common-app-lib/src/cmr/common_app/services/search/elastic_search_index.clj

```diff
@@ -46,6 +46,12 @@
   (fn [concept-type query]
     [concept-type (qm/base-result-format query)]))
 
+(defmethod concept-type+result-format->fields [:granule :xml]
```

Comment:
I have not seen a + sign in a function name in our code base, have you seen another function do this? Obviously you can do it, but it stands out and I wonder what other developers think.